superset/views/dashboard/views.py

```diff
@@ -86,7 +85,11 @@ def new(self) -> FlaskResponse:
         )
         db.session.add(new_dashboard)
         db.session.commit()  # pylint: disable=consider-using-transaction
-        return redirect(f"/superset/dashboard/{new_dashboard.id}/?edit=true")
+        return redirect(
+            url_for(
+                "Superset.dashboard", dashboard_id_or_slug=new_dashboard.id, edit=True
```

Comment:
Somehow this line broke some tests on `master` but I don't understand how it didn't get caught by CI (????).

Issues is with the capitalize `True` here... Maybe you IDE did this? Linter?